Lib/test/test_itertools.py

```diff
@@ -148,15 +146,19 @@ def test_accumulate(self):
                          [2, 16, 144, 720, 5040, 0, 0, 0, 0, 0])
         with self.assertRaises(TypeError):
             list(accumulate(s, chr))                                # unary-operation
-        for proto in range(pickle.HIGHEST_PROTOCOL + 1):
-            self.pickletest(proto, accumulate(range(10)))           # test pickling
-            self.pickletest(proto, accumulate(range(10), initial=7))
         self.assertEqual(list(accumulate([10, 5, 1], initial=None)), [10, 15, 16])
         self.assertEqual(list(accumulate([10, 5, 1], initial=100)), [100, 110, 115, 116])
         self.assertEqual(list(accumulate([], initial=100)), [100])
         with self.assertRaises(TypeError):
             list(accumulate([10, 20], 100))
 
+    # TODO: RUSTPYTHON
+    @unittest.expectedFailure
+    def test_accumulate_pickle(self):
```

Comment:
You might want to take this change up with the CPython devs.

Reply:
Refer to the new commits.